### DISCUSSION TITLE
Why "OK" is lemmatized as "ok" while it is usually spelled as all capital in most dictionaries? And "people" at the beginning of a sentence is lemmatized as "People"

### DISCUSSION THREAD

**Discussion by zhanqiwang** [explosion/spaCy#6558] [Help: Other Questions]
## How to reproduce the behaviour

```
import spacy
nlp = spacy.load("en_core_web_sm", disable=["ner"])
s = "OK! What're your names? It's OK to say No. People like flowers."
doc = nlp(s)
print("\n".join(t.lemma_ for t in doc))
```
## Info about spaCy
* **spaCy version:** 2.3.4
* **Platform:** Windows-10-10.0.19041-SP0
* **Python version:** 3.8.6

**Comment by svlandeg** [comment]
FYI - we converted this question to a thread on the discussion board. 

We're aiming to keep the Issue tracker focused mainly on bug reports, while the new GH Discussion board can be a place where the community can come together, help eachother, discuss best practices, and so on.
